VaccineCard/VaccineCard/Extensions & utilities/Data.swift

```diff
@@ -13,21 +13,22 @@ extension Data {
                 let decompressedData: NSData = try (self as NSData).decompressed(using: .zlib)
                 guard let string = String(data: decompressedData as Data, encoding: .utf8),
                       let data = string.data(using: .utf8) else {
-                    print("Failed while decompressing data")
+                    print("Failed while decompressing data") // TODO: Better error handling - show user?
                     return nil
```

Comment:
Should this be something to bring up at standup and see what the team thinks in terms of a user friendly message?